lib/chef/server.rb

```diff
@@ -23,8 +23,7 @@ def parse_server_version(line)
 
     def version
       @version ||= begin
-                     uri = URI.parse("#{root_url}/version")
-                     ver_line = open(uri, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE}).each_line.first
+                     ver_line = Chef::ServerAPI.new(root_url).get('version').each_line.first
```

Comment:
Does this mean we'll be verifying the TLS cert by default now?  I think that is OK, but it may require some people to update their processes.

Reply:
@stevendanna interesting point, however when I think about your question I don't think I'm concerned. This one location isn't the first or only place where we're using `Chef::ServerAPI`. Therefore, unless I'm mistaken, I don't believe this will have any net change in how the client is verifying certs when interrogating the Chef Server APIs. 

```
✔ ~/Devel/ChefProject/knife-ec-backup [jeremymv2/sql_db_option L|…8⚑ 7]
13:27 $ grep -r Chef::ServerAP lib/
lib//chef/server.rb:                     ver_line = Chef::ServerAPI.new(root_url).get('version').each_line.first  # this is the line I added 
lib//chef/server.rb:      Chef::ServerAPI.new("http://127.0.0.1:9465").get("users")
lib//chef/knife/ec_base.rb:          rest = Chef::ServerAPI.new(Chef::Config.chef_server_url, {:api_version => "0"})
lib//chef/knife/ec_base.rb:        @rest ||= Chef::ServerAPI.new(server.root_url, {:api_version => "0"})
lib//chef/knife/ec_base.rb:                             Chef::ServerAPI.new("http://127.0.0.1:9465", {:api_version => "0"})
✔ ~/Devel/ChefProject/knife-ec-backup [jeremymv2/sql_db_option L|…8⚑ 7]
13:30 $
```

Reply:
The argument here is that all the other API endpoints are accessed differently (SSL verification turned on), this one is an outlier for no clear reason